frontend/src/utils/formatting/string.ts

```
@@ -0,0 +1,162 @@
+ * @param decimals - Optional. The number of decimal places to include in the formatted output. Defaults to 2.
+ * @returns A string representing the formatted file size with units.
+ *
+ * @example
+ * formatFileSize(1536); // "1.5 KB"
+ * formatFileSize(1048576, 1); // "1.0 MB"
+ * formatFileSize(0); // "0 Bytes"
+ *
+ * @since 1.0.0
+ * Generated by CodeRabbit
+ */
+export function formatFileSize(bytes: number, decimals: number = 2): string {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)